### DISCUSSION TITLE
Pleas Help curl error 60 while downloading https://codeload.github.com

### DISCUSSION THREAD

**Discussion by rimadjamaa** [community/community#63653] [New to GitHub] [answered]
when i try to create a new laravel project i face this issues  :
```
Failed to download spatie/flare-client-php from dist: curl error 28 while downloading https://codeload.github.com/spatie/flare-client-php/legacy.zip/5f2c6a7a0d2c1d90c12559dc7828fd942911a544: Operation timed out after 16956 milliseconds with 0 out of 0 bytes received
    Now trying to download from source
```
composer can not access to HTTPS github files 
1.The issue is consistent across different browsers.

2.Changing DNS settings did not resolve the problem.

3.Temporarily disabling SSL in the browser did not resolve the issue.

4.The problem seems to be related to SSL/TLS communication between my computer and the server.

can any one help me to resolve this issue ?

**Comment by queenofcorgis** [comment, marked as answer]
If you're asking how to use the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project), if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.

**Reply by rimadjamaa** [reply]
No not asking how to use the piece of software .I am encountering an issue with Composer while attempting to create a new Laravel project using the following command:
`composer create-project laravel/laravel example-app`
Upon execution, I am faced with several cURL 60 errors that are preventing Composer from accessing GitHub packages due to SSL certification problems. Unfortunately, there are a significant number of these errors, which makes it challenging to provide them all. You can see a snapshot of the error messages in the screenshot below:
<img width="958" alt="image" src="https://github.com/community/community/assets/88832599/0fb132ec-01c2-4dd4-9c12-f06bd7ef4b5e">
To troubleshoot the issue, I verified the SSL certificate using OpenSSL by running the following command:
`$ openssl s_client -connect codeload.github.com:443`
The SSL certificate appears to be valid and correctly signed by a trusted CA, as shown in the output of the command:
```
Certificate chain
 0 s:CN = dns.google
   i:C = US, O = Google Trust Services LLC, CN = GTS CA 1C3
...
Server certificate
-----BEGIN CERTIFICATE-----
...
-----END CERTIFICATE-----
subject=CN = dns.google
issuer=C = US, O = Google Trust Services LLC, CN = GTS CA 1C3
...
Verify return code: 0 (ok)

```
Despite the valid SSL certificate, Composer is still encountering SSL certification issues.

I have already attempted a few solutions, such as clearing the Composer cache,changing DNS,enabling SSL of website ,Unable firewall updating Git, and trying different installation directories. However, the issue persists.

**Comment by luisgv01** [comment]
Hola. Solucioné el problema agregando esta dirección web (https://codeload.github.com) a las exclusiones de mi antivirus (Norton)

**Reply by v-coder247** [reply]
Hi @luisgv01 , thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting to ensure discussions are adhering to our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md).
